src/v2/controllers/FormController.js

```diff
@@ -153,10 +153,15 @@ export default Controller.extend({
       // TODO: OKTA-243167 what's the approach to show spinner indicating API in flight?
       actionFn()
         .then((resp) => {
-          if (actionPath === 'cancel' && this.options.settings.get('useInteractionCodeFlow')) {
-            // In this case we need to restart login flow and recreate transaction meta
-            // that will be used in interactionCodeFlow function
-            this.options.appState.trigger('restartLoginFlow');
+          const { settings, appState } = this.options;
+          if (actionPath === 'cancel') {
```

Comment:
on `cancel` we never hit `this.handleIdxResponse` now.. seems risky change to me. Just verify once in bacon first.

Reply:
Yea, and it would clear any url parameters which may be needed.. I pushed primarily so I could test this in okta-core.

Reply:
SIW will call what `handleIdxResponse` does when we cancel because of the page reload